gef-ui/server/controllers/about-facility/index.validate-and-update-about-facility.test.js

```diff
@@ -109,6 +109,8 @@ describe('validateAndUpdateAboutFacility', () => {
       };
       if (dealVersion === 1) {
         expectedPayload.isUsingFacilityEndDate = true;
+        expectedPayload.facilityEndDate = null;
+        expectedPayload.bankReviewDate = null;
```

Comment:
Can you add a test to this file to unit test the logic in the `index.js` file - where if the update has e.g. a facility end date but also the cover start date has changed, then it shouldn't return the facility end date values?

Reply:
This file won't update the `facilityEndDate` because you cant update it on the `about-facility` page (If i've understood correctly)

Reply:
This test name feels out of date with what we're doing -- is there a shuffle (or even just a new explicit test) to check we're wiping FED and BRD if CED changes (TLA)

Reply:
Sorry, revisiting the comment, we probably should have unit tests checking the behaviour of when were setting Cover end date and that it wipes FED/BRD just for documentation purposes (and that it doesnt wipe these when cover end date doesnt change)